build/buildBuildImages.sh

```diff
@@ -175,16 +178,13 @@ function buildGitHubActionsImage() {
 	local devImageTag=github-actions
 	local builtImageName="$ACR_BUILD_GITHUB_ACTIONS_IMAGE_NAME"
 
-	if [ -z "$debianFlavor" ] || [ "$debianFlavor" == "stretch" ]; then
+	if [ -z "$debianFlavor" ] ; then
 		debianFlavor="stretch"
-		echo "Debian Flavor is: "$debianFlavor
-	elif [ "$debianFlavor" == "buster" ] || [ "$debianFlavor" == "bullseye" ]; then
-		debianFlavor=$debianFlavor
-		devImageTag=$devImageTag-$debianFlavor
-		echo "dev image tag: "$devImageTag
-		builtImageName=$builtImageName-$debianFlavor
-		echo "built image name: "$builtImageName
 	fi
+	devImageTag=$devImageTag-debian-$debianFlavor
```

Comment:
I'm wondering if it's good place to hardcode `debian` here, as we also have `ubuntu` OS type, so when $debianFlavor is `focal-scm`, it would tag wrongly?
https://github.com/microsoft/Oryx/blob/0405a71cca5640a8d2bb41fa7a234271989b09dc/src/BuildScriptGenerator.Common/OsTypes.cs#L10
Another place I think we'll probably address is here in vso focal dockerfile we have `ENV DEBIAN_FLAVOR="focal-scm"` ->` ENV UBUNTU_FLAVOR="focal-scm"` or simply rename all `DEBIAN_FLAVOR` to `OS_FLAVOR`? 
https://github.com/microsoft/Oryx/blob/0405a71cca5640a8d2bb41fa7a234271989b09dc/images/build/Dockerfiles/vso.focal.Dockerfile#L72

Reply:
Are we planning on releasing an ubuntu github actions image? Since this is taking debian flavor as an input, I think that it is ok to keep here hardcoded as debian. I only say this since it seems to fall under a larger effort that you mentioned, and we hardcode "Debian" throughout our repository.

But yeah its a super good point that we are essentially using DEBIAN_FLAVOR as a synonym to OS_TYPE in certain places in our code which creates confusion. It would be valuable to the longevity of our code to rename debian flavor to os type, but I think that we should have a ticket devoted solely to that work. It would likely be good to do before the introduction of the Mariner OS type?